File: src/digivance_auth/packages/apiclient/commands/UpdateTenant.ts
/**
 * Command to update an existing tenant
 */
export default interface UpdateTenant {
    /**
     * User friendly description of this tenant
     */
    description?: string;

    /**
     * Unique id of the tenant we want to update
     */
    id: string; // Actually a Guid

    /**
     * User friendly name of this tenant
     */
    name: string;
};

File: src/Digivance.Auth.Data/Commands/UpdateTenant.cs
namespace Digivance.Auth.Data.Commands
{
    public record UpdateTenant
    {
        /// <summary>
        /// User friendly description of this tenant
        /// </summary>
        public string Description { get; set; }

        /// <summary>
        /// Unique id of the tenant to update
        /// </summary>
        public Guid Id { get; set; }

        /// <summary>
        /// User friendly name of this tenant
        /// </summary>
        public string Name { get; set; }
    }
}
